frontend/web-app/src/Components/Chat/chatObserve.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState} from "react";
+import socketioclient from "socket.io-client";
+
+function ChatObservation(){
+    
+    const [queue, setQueue] = useState([0]);
+
+    
+
+    useEffect(()=>{
+        let socket = socketioclient("http://localhost:7000");
```

Comment:
I think it is a good idea to start making the addresses into variables instead of hardcoding them,
making it easier for deployment and maintaining later on

Reply:
Agreed on that!